client/src/components/aggregations/utils.tsx

```diff
@@ -241,27 +218,22 @@ export function reportsToEvents(reports, showInterlocutors) {
         // and finally ascending by title
         r1.title.localeCompare(r2.title)
     )
+
+  // Do we have an event to show as well?
+  if (event) {
+    result.push(createCalendarEventFromEvent(event))
+  }
```

Comment:
This makes the order of the calendar events shown different: on the EventSeries, you show events then reports, on the Event, you show reports then the event. I would expect the order to be the same in both cases.